docs/syntax-cheatsheet.md

```diff
@@ -2,4 +2,210 @@
 title: Syntax Cheatsheet
 ---
 
-This section has been moved to the new ReScript project: https://rescript-lang.org/docs/manual/v8.0.0/overview. More info on the transition [here](https://rescript-lang.org/blog/bucklescript-is-rebranding).
+Here is the cheat sheet with some equivalents between JavaScript and Reason syntaxes:
+
+### Variable
+
+| JavaScript              | Reason                         |
+|-------------------------|--------------------------------|
+| `const x = 5;`          | `let x = 5;`                   |
+| `var x = y;`            | No equivalent                  |
+| `let x = 5; x = x + 1;` | `let x = ref(5); x := x^ + 1;` |
+
+### String & Character
+
+| JavaScript             | Reason                |
+|------------------------|-----------------------|
+| `"Hello world!"`       | Same                  |
+| `'Hello world!'`       | Strings must use `"`  |
+| Characters are strings | `'a'`                 |
+| `"hello " + "world"`   | `"hello " ++ "world"` |
+
+### Boolean
+
+| JavaScript                                            | Reason                            |
+|-------------------------------------------------------|-----------------------------------|
+| `true`, `false`                                       | Same                              |
+| `!true`                                               | Same                              |
+| `||`, `&&`, `<=`, `>=`, `<`, `>`                      | Same                              |
+| `a === b`, `a !== b`                                  | Same                              |
+| No deep equality (recursive compare)                  | `a == b`, `a != b`                |
+| `a == b`                                              | No equality with implicit casting |
+
+### Number
+
+| JavaScript  | Reason                |
+|-------------|-----------------------|
+| `3`         | Same \*               |
+| `3.1415`    | Same                  |
+| `3 + 4`     | Same                  |
+| `3.0 + 4.5` | `3.0 +. 4.5`          |
+| `5 % 3`     | `5 mod 3`             |
+
+\* JavaScript has no distinction between integer and float.
+
+### Object/Record
+
+| JavaScript          | Reason                                  |
+|---------------------|-----------------------------------------|
+| no static types     | `type point = {x: int, mutable y: int}` |
+| `{x: 30, y: 20}`    | Same                                    |
+| `point.x`           | Same                                    |
+| `point.y = 30;`     | Same                                    |
+| `{...point, x: 30}` | Same                                    |
+
+### Array
+
+| JavaScript            | Reason                             |
+|-----------------------|------------------------------------|
+| `[1, 2, 3]`           | <code>\[\|1, 2, 3\|\]</code> |
```

Comment:
This is borked

<img width="552" alt="image" src="https://github.com/reasonml/reasonml.github.io/assets/220424/92850378-d68d-4bf3-9214-2cc92bb08937">